var/spack/repos/builtin/packages/root/loader.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/cmake/modules/RootBuildOptions.cmake b/cmake/modules/RootBuildOptions.cmake
```

Comment:
The file var/spack/repos/builtin/packages/root/loader.patch isn't used anymore. Once removed we can move ahead and merge here.